external/storm-hdfs/src/test/java/org/apache/storm/hdfs/testing/MiniDFSClusterExtensionClassLevel.java

```diff
@@ -52,6 +55,8 @@ public MiniDFSCluster getDfscluster() {
     public void beforeAll(ExtensionContext arg0) throws Exception {
         System.setProperty(TEST_BUILD_DATA, "target/test/data");
         hadoopConf = hadoopConfSupplier.get();
+        String tempDir = getTestDir("dfs").getAbsolutePath() + File.separator;
```

Comment:
is this somehow related to junit changes or is this some cleanup to use a test dir?

Reply:
Thanks for the explanation